tools/flaky_test_monitor/process_results.go

```diff
@@ -49,12 +49,13 @@ func (testRun *TestRun) save(fileName string) {
 
 // models test result of an entire package which can have multiple tests
 type PackageResult struct {
-	Package string                  `json:"package"`
-	Result  string                  `json:"result"`
-	Elapsed float32                 `json:"elapsed"`
-	Output  []string                `json:"output"`
-	Tests   []TestResult            `json:"tests"`
-	TestMap map[string][]TestResult `json:"-"`
+	Package  string                  `json:"package"`
+	Result   string                  `json:"result"`
+	Elapsed  float32                 `json:"elapsed"`
+	Output   []string                `json:"output"`
+	Tests    []TestResult            `json:"tests"`
+	NilTests []TestResult            `json:"nil_tests"`
```

Comment:
Possibly, I'm open to it - but since the 2nd layer script (summary computation) isn't done yet, we need to store nil tests somewhere. For now, I think we should leave it here and possibly move it to the summary script.